README.md

```diff
@@ -22,7 +22,7 @@ Manage todo lists with ease. Powerful, easy to use and customizable. [View the d
 - **[TaskPaper](https://www.taskpaper.com) compatible**: just set `todo.symbols.box`, `todo.symbols.done` and `todo.symbols.cancelled` to `-`
 - **Timekeeping**: you can mark todos as started and track elapsed time until completion
 - **Timer**: a timer can be displayed in the statusbar for started todos
-- **Time estimates**: you can estimate the time it will take to complete a todo by adding a tag to it that looks like this: `@est(3 hours)`, `@est(2h30m)` or `@2h30m`. Then you can use the `[est]` token in statistics
+- **Time estimates**: you can estimate the time it will take to complete a todo by adding a tag to it that looks like this: `@est(3 hours)`, `@est(2h30m)` or `@2h30m`. Then you can use the `[est]` and `[est-total]` token in statistics
```

Comment:
It should be "token**s**".

Reply:
Nice catch! Will update those the upcoming days :) Looking forward to using this

Reply:
See newest commit